configure.ac

```diff
@@ -120,6 +120,9 @@ AC_CHECK_FUNC([timegm], [], [AC_MSG_ERROR([timegm() function not found])])
 # Check for endian headers
 AC_CHECK_HEADERS([endian.h sys/endian.h byteswap.h])
 
+# Check for PAM extensions header
+AC_CHECK_HEADERS([security/pam_ext.h])
```

Comment:
This is already checked for:
https://github.com/SSSD/sssd/blob/8c3074a97b4a80d74f8d7a8fc3b6e0d8e0d12fb1/src/external/pam.m4#L11

Reply:
And then used in `https://github.com/SSSD/sssd/blob/master/src/sss_client/sss_pam_compat.h`

Reply:
But that's weird `sss_pam_compat.h` doesn't include "config.h"
Imo, it's a bug.